config/manager/network_policy.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-ingress-https-only-and-block-egress
+  namespace: system
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - ports:
+    - protocol: TCP
+      port: 443,444,8080,8443,4443,9443
```

Comment:
Sort them, please. Also, this is missing the health probe port. Is this on purpose? Regarding the rest of the ports? Why? Do we need them?